src/test/java/homeWorks/ttykhovetska/HandlingExceptionsHomeWork.java

```diff
@@ -0,0 +1,63 @@
+package homeWorks.ttykhovetska;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+import static java.util.Arrays.asList;
+
+public class HandlingExceptionsHomeWork {
+
+    public static void main(String[] args) {
+
+        List<String> fruits = new ArrayList<String>(asList("banana", "kiwi", "orange", "watermelon", "grape"));
+        getListElements(fruits);
+    }
+
+
+    public static void getListElements(List<String> inputList){
+        try {
+            for (int i = 0; i <= inputList.size(); i++) {
+                System.out.println(inputList.get(i));
+            }
+        }
+        catch (IndexOutOfBoundsException e) {
+            System.out.println("There are no fruits. Buy more.");
+        }
+
+    }
+
+    public static void getTextFile(){
+        try {
+            FileInputStream fr= new FileInputStream("~/no_such_file.txt");
+        }
+        catch (FileNotFoundException e){
+            System.out.println("There is no file with such path");
+        }
+
+    }
+
+    public static void numberFormatException() {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter any valid number");
+        try {
+            Integer.parseInt(scanner.next());
+        } catch (NumberFormatException e) {
+            System.out.println("Programme end");
+        }
+    }
```

Comment:
according to the issue "Programme end" message need to show all the time in console, but if you fill number (valid value) I can see the message. Add finally block your method